src/main/resources/google_checks.xml

```diff
@@ -95,6 +95,12 @@
              value="CLASS_DEF, METHOD_DEF, CTOR_DEF, LITERAL_FOR, LITERAL_WHILE, STATIC_INIT,
                     INSTANCE_INIT, ANNOTATION_DEF, ENUM_DEF"/>
         </module>
+        <module name="SuppressionXpathSingleFilter">
+          <!-- suppresion is required till https://github.com/checkstyle/checkstyle/issues/7541 -->
+          <property name="id" value="RightCurlyAlone"/>
+          <property name="query" value="//RCURLY[parent::SLIST[count(./*)=1]
+                                                 or parent::OBJBLOCK[count(./*)=2]]"/>
```

Comment:
This won't work with comments as specified in another issue.

Reply:
For Google style it ok. It will work as required.